tests/ruby-sass/media-width.scss

```diff
@@ -0,0 +1,37 @@
+@import "../setup";
+
+.media-width-from {
+  @include wtr-media-width(from, 250px) {
+    color: red
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Declaration should be terminated by a semicolon